src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -300,6 +301,46 @@ public PersistentDataStore() {
 		return activities;
 	}
 
+  /**
+   * Loads all Profile objects from the Datastore service and returns them in a List, sorted in
+   * ascending order by creation time.
+   * @throws PersistentDataStoreException if an error was detected during the load from the
+   *			Datastore service
+   */
+
+  public List<Profile> loadProfiles() throws PersistentDataStoreException {
+  List<Profile> profiles = new ArrayList<>();
+
+    // Retrieve all activities from the datastore.
+    Query query = new Query("chat-profiles").addSort("creation_time", SortDirection.DESCENDING);
+    PreparedQuery results = datastore.prepare(query);
+
+    for(Entity entity : results.asIterable()) {
+      try {
+        UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
+        Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
+        // UUID picId = UUID.fromString((String) entity.getProperty("pic_id"));
```

Comment:
Why is this commented out?

Reply:
Because as of right now, the only thing I am saving in Profile.java is the UUID, an Instant, and then I check to see if there has been an aboutMe added to the the object by the user, I will uncomment these out as I add each feature to profiles.